en/docs/develop/creating-artifacts/using-docker-secrets.md

```diff
@@ -87,14 +90,8 @@ Given below is a sample synapse configuration that accesses and prints the file
 
 ## Enabling secrets in the environment
```

Comment:
https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-titles - Use sentence case and avoid `ing`